index.html

```diff
@@ -919,13 +918,285 @@ <h4>Verify Proof (eddsa-jcs-2022)</h4>
+                <li>
+Check that the |securedDocument|.<var>@context</var> starts with all values
+contained in the |proofOptions|.<var>@context</var> in the same order.
+Otherwise, set |verified| to `false` and skip to the last step.
+                </li>
+                <li>
+Set |unsecuredDocument|.<var>@context</var> equal to
+|proofOptions|.<var>@context</var>.
+                </li>
+              </ol>
+            </li>
+            <li>
+Let |transformedData| be the result of running the algorithm in Section
+<a href="#transformation-eddsa-jcs-2022"></a> with |unsecuredDocument| and
+|proofOptions| passed as parameters.
+              </li>
+            <li>
+Let |proofConfig| be the result of running the algorithm in
+Section [[[#proof-configuration-eddsa-jcs-2022]]] with
+|unsecuredDocument| and |proofOptions| passed as a parameters.
+            </li>
+            <li>
+  Let |hashData| be the result of running the algorithm in Section
+  [[[#hashing-eddsa-jcs-2022]]] with |transformedData| and |proofConfig|
+  passed as a parameters.
+            </li>
+            <li>
+  Let |verified:boolean| be the result of running the algorithm in Section
+  [[[#proof-verification-eddsa-jcs-2022]]] algorithm on |hashData|,
+  |proofBytes|, and |proofConfig|.
+              </li>
+              <li>
+  Return a [=verification result=] with [=struct/items=]:
+                <dl data-link-for="verification result">
+                  <dt>[=verified=]</dt>
+                  <dd>|verified|</dd>
+                  <dt>[=verifiedDocument=]</dt>
+                  <dd>
+  |unsecuredDocument|, if |verified| is `true`; otherwise, <a data-cite="INFRA#nulls">Null</a></dd>
+                </dl>
+              </li>
+            </ol>
+        </section>
+        <section>
+          <h4>Transformation (eddsa-jcs-2022)</h4>
+
+          <p>
+The following algorithm specifies how to transform an unsecured input document
+into a transformed document that is ready to be provided as input to the
+hashing algorithm in Section <a href="#hashing-eddsa-jcs-2022"></a>.
+          </p>
+
+          <p>
+Required inputs to this algorithm are an
+<a data-cite="vc-data-integrity#dfn-unsecured-data-document">
+unsecured data document</a> (<var>unsecuredDocument</var>) and
+transformation options (<var>options</var>). The
+transformation options MUST contain a type identifier for the
+<a data-cite="vc-data-integrity#dfn-cryptosuite">
+cryptographic suite</a> (<var>type</var>) and a cryptosuite
+identifier (<var>cryptosuite</var>). A <em>transformed data document</em> is
+produced as output. Whenever this algorithm encodes strings, it MUST use UTF-8
+encoding.
+          </p>
+
+          <ol class="algorithm">
+            <li>
+If <var>options</var>.<var>type</var> is not set to the string
+`DataIntegrityProof` and <var>options</var>.<var>cryptosuite</var> is not
+set to the string `eddsa-jcs-2022` then a `PROOF_TRANSFORMATION_ERROR` MUST be
+raised.
```

Comment:
Errors are MUSTs, specific error codes are SHOULDs, and `PROOF_TRANSFORMATION_ERROR` isn't defined anymore, so I picked `MALFORMED_PROOF_ERROR`:

```suggestion
set to the string `eddsa-jcs-2022` then an error MUST be raised that
SHOULD use the `MALFORMED_PROOF_ERROR` error code.
```